boards/Kconfig

```
@@ -370,7 +370,16 @@ config ARCH_BOARD_ESP32S2_SAOLA_1
 	select ARCH_HAVE_IRQBUTTONS if ESP32S2_GPIO_IRQ
 	---help---
 		This is the ESP32-S2-Saola-1 board
-
+		
+config ARCH_BOARD_ESP32S2_FRANZININHO
```

```suggestion
config ARCH_BOARD_FRANZININHO_WIFI
```
Rename `config` for consistency.